index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Personal Portfolio</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+</head>
+
+<body>
+
+  <header>
+    <div class="wrapper menu">
+      <span class="menu-logo">My Logo</span>
```

Comment:
Great job on the UI! I noticed that you have a logo text displayed on the left side of the page. However, it's important to note that the logo should typically be wrapped in an `<a>` tag to provide a link to the home page.

:pray: Please consider implementing a link (`<a>` tag) for your logo to enhance the overall user experience and adhere to established conventions.  Check [this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.